src/main/java/org/zalando/nakadi/filters/LoggingFilter.java

```diff
@@ -164,34 +166,46 @@ private void logToAccessLog(final RequestLogInfo requestLogInfo, final int statu
     }
 
     private void traceRequest(final RequestLogInfo requestLogInfo, final int statusCode, final Long timeSpentMs) {
-        if (requestLogInfo.path != null && "POST".equals(requestLogInfo.method) &&
-                requestLogInfo.path.startsWith("/event-types/") && requestLogInfo.path.contains("/events")) {
+        if (!isPublishingRequest(requestLogInfo)) {
+            return;
+        }
+        final String eventType = requestLogInfo.path.substring("/event-types/".length(),
+                requestLogInfo.path.lastIndexOf("/events"));
 
-            final String eventType = requestLogInfo.path.substring("/event-types/".length(),
-                    requestLogInfo.path.lastIndexOf("/events"));
+        String sloBucket = "5K-50K";
+        if (requestLogInfo.contentLength < 5000) {
```

Comment:
Shouldn't content length of 0 be excluded from here?

Reply:
very true! Fixed https://github.com/zalando/nakadi/pull/1022/commits/efe8bf4307bd4fc147c0914fcf7c27799eda6234